object.go

```diff
@@ -292,36 +295,71 @@ func (d *Decoder) decClassDef() (interface{}, error) {
 	return classInfo{javaName: clsName, fieldNameList: fieldList}, nil
 }
 
-func findField(name string, typ reflect.Type) ([]int, error) {
+type fieldInfo struct {
+	indexes []int
+	field   reflect.StructField
+}
+
+// map[rType][fieldName]indexes
+var fieldIndexCache sync.Map
+
+func findField(name string, typ reflect.Type) (indexes []int, field reflect.StructField, err error) {
```

Comment:
in dubbo-go/dubbo-go-hessian2, we do not suggest set the var name for the return value. pls delete the return value name.